src/home/home.html

```diff
@@ -51,7 +51,7 @@
     <div class="section fairLaunch">
       <div class="pic"><img src="/spacex-OHOU-5UVIYQ-unsplash.png" /></div>
       <div class="rightColumn">
-        <div class="title heading heading1">The real fair launch</div>
+        <div class="title heading heading3">The real fair launch</div>
```

Comment:
please just override the css for heading1 rather than creating in home.scss a heading3 (which also exists in styles.scss)